CODEOWNERS

```diff
@@ -0,0 +1,309 @@
+.asf.yml @apache/geode-committers
+ 
+#-----------------------------------------------------------------
+# Serialization
+#-----------------------------------------------------------------
+geode-serialization/**                                            @bschuchardt @Bill
+geode-core/**/org/apache/geode/pdx/**                             @upthewaterspout @dschneider-pivotal
+geode-core/**/org/apache/geode/internal/*                         @bschuchardt @Bill
+
+#-----------------------------------------------------------------
+# Membership
+#-----------------------------------------------------------------
+geode-membership/**                                               @bschuchardt @Bill
+geode-old-client-support/**                                       @bschuchardt @Bill
+geode-tcp-server/**                                               @bschuchardt @Bill
+geode-core/**/org/apache/geode/distributed/internal/membership/** @bschuchardt @Bill
+
+#-----------------------------------------------------------------
+# P2P Messaging
+#-----------------------------------------------------------------
+geode-core/**/org/apache/geode/internal/tcp/**                    @bschuchardt @Bill
+geode-core/**/org/apache/geode/distributed/internal/direct/**     @bschuchardt @Bill
+geode-core/**/org/apache/geode/internal/net/**                    @bschuchardt @Bill
+geode-core/**/org/apache/geode/net/**                             @bschuchardt @Bill
+geode-core/**/org/apache/geode/distributed/*                      @bschuchardt @Bill
+geode-core/**/org/apache/geode/distributed/internal/*             @bschuchardt @Bill
+
+#-----------------------------------------------------------------
+# Client/server messaging and cache operations
+#-----------------------------------------------------------------
+geode-core/**/org/apache/geode/cache/client/**                    @bschuchardt @Bill
+geode-core/**/org/apache/geode/cache/server/**                    @bschuchardt @Bill
+geode-core/**/org/apache/geode/internal/cache/client/**           @bschuchardt @echobravopapa
+geode-core/**/org/apache/geode/internal/cache/tier/**             @bschuchardt @echobravopapa
+
+#-----------------------------------------------------------------
+# Client Queues
+#-----------------------------------------------------------------
+geode-core/**/org/apache/geode/internal/cache/ha/**               @agingade @boglesby
+geode-core/**/org/apache/geode/internal/cache/tier/CacheClient*   @agingade @boglesby
+ 
+#-----------------------------------------------------------------
+# Experimental Protobuf based Client/Server
+#-----------------------------------------------------------------
+geode-experimental-driver/**                                      @kohlmu-pivotal @PurelyApplied
+geode-protobuf/**                                                 @kohlmu-pivotal @PurelyApplied
+geode-protobuf-messages/**                                        @kohlmu-pivotal @PurelyApplied
```

Comment:
Not sure if I'm the best person to be a codeowner for code that I haven't looked at in the longest time. @upthewaterspout or @galen-pivotal would be MUCH better alternatives for this.